packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -6,6 +6,41 @@ import 'package:meta/meta.dart';
 
 import 'print.dart';
 
+/// The various priority levels used to filter which diagnostics are shown and
+/// omitted.
+///
+/// Trees of Flutter diagnostics can be very large so filtering the Diagnostics
```

Comment:
if Diagnostics is a reference to an object, link to it; otherwise, lowercase d.

Reply:
done